README.md

```diff
@@ -35,6 +35,7 @@ This will produce an XML string, and pipe it directly into the file called packa
 |---|---|
 |-a, --api|Set the API version of the packagexml file (Default is 42.0)|
```

Comment:
@jayree can you update this line to say the default is now 43?

Reply:
done